consensus/ethash/consensus.go

```diff
@@ -581,6 +587,46 @@ func accumulateRewards(config *params.ChainConfig, state *state.StateDB, header
 	if config.IsEthersocial(header.Number) {
 		blockReward = EthersocialBlockReward
 	}
+	if config.IsMCIP0(header.Number) {
```

Comment:
[The way go-musicoin implements this](https://github.com/Musicoin/go-musicoin/blob/master/consensus/ethash/consensus.go#L537) is not that pretty; there is even an unintended "legacy uncle reward" b/c of buggy code. So instead of trying to understand and refactor, I just chose the copy-paste opt-in pattern.

Reply:
Extract this calculation to a separate method.

Optionally - put the new method into separate file? (for easier merging with upstream?) (There will be change in this file anyways...)